warp/cli/warp_cli.py

```diff
@@ -77,7 +80,7 @@ def transpile(file_path, contract_name, cairo_output):
     ),
 )
 @click.pass_obj
-def invoke(obj, program, address, function, inputs):
+def call(obj, program, address, function, inputs, gateway_url):
```

Comment:
Isn't `gateway_url` already passed from `obj`? This will be a duplicated parameter